src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -109,8 +109,8 @@
 
   <!-- Common dependencies across all targets -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.*" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.*" />
+	<PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.1" />
+	<PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.1" />
```

Comment:
nit: replace tabs with spaces

Reply:
Done

Reply:
This still shows as tabs for me.

Reply:
Hmm. I tried again. Should have worked this time.